test/cmdline_test.py

```diff
@@ -135,35 +144,26 @@ def test_cmdline_ambiguous_class(self, logger):
     @mock.patch("logging.getLogger")
     @mock.patch("logging.StreamHandler")
     def test_setup_interface_logging(self, handler, logger):
+        opts = type('opts', (), {})
+        opts.background = False
+        opts.logdir = False
+        opts.logging_conf_file = None
+        opts.log_level = 'INFO'
+
         handler.return_value = mock.Mock(name="stream_handler")
-        with mock.patch("luigi.interface.setup_interface_logging.has_run", new=False):
-            luigi.interface.setup_interface_logging()
-            self.assertEqual([mock.call(handler.return_value)], logger.return_value.addHandler.call_args_list)
-
-        with mock.patch("luigi.interface.setup_interface_logging.has_run", new=False):
-            if six.PY2:
-                error = ConfigParser.NoSectionError
-            else:
-                error = KeyError
-            self.assertRaises(error, luigi.interface.setup_interface_logging, '/blah')
-
-    @mock.patch("warnings.warn")
-    @mock.patch("luigi.interface.setup_interface_logging")
-    def test_cmdline_logger(self, setup_mock, warn):
```

Comment:
It's so much code that it's hard to review it all. But can you make sure that all tests you delete are added somewhere else?

Reply:
Yes, all logging well tested in `test/setup_logging_test.py`